### ISSUE TITLE
Cyborgs should have more door interactions

### ISSUE THREAD

**Issue by Kayek14** [space-wizards/space-station-14#21127]
## Description
Cyborgs could use more ways of interacting with doors. They should be able to remotly open, bolt, and shock doors or set emegrency access. iirc in SS13 ctrl click bolted a door, alt click electrfied it, shift click remote opened it (if borg had access)

i would mark this Feature Request but i dont know how.

**Comment by NancokPS2** [comment]
The shocking would need to be syndie only, since there is no reason for a borg to have the ability to do so otherwise.  
Opening and bolting doors remotely would probably lead to shitters being shitters without much chance for crew to tell who is the culprit. This kinda stuff is rampant on LRP and something staff are very much against making easier.

**Comment by Kayek14** [comment]
Bolting doors is an important tool for borgs to use to prevent crew harm. If any borg is going around doing it just to be a shitter they should be ahelped.

**Comment by IProduceWidgets** [comment]
I disagree about the emag thing because there are (and ideally will be) more ways for laws to change.
